### PR TITLE
Updates to argo to include an Allow header with a 405 error.

### PR DESCRIPTION
**Pull request by mdobson**
This is a fix for #50

### CODE REVIEW

File: test/argo_test.js [hunk 1]
```diff
@@ -968,12 +968,13 @@ describe('Argo', function() {
   });
 
   describe('method routing', function() {
-    it('returns a 405 Method Not Allowed on unsupported methods', function(done) {
+    it.only('returns a 405 Method Not Allowed on unsupported methods', function(done) {
```
Comment (AdamMagaluk) [inline test/argo_test.js]: Gotta get rid of this.

**Comment by AdamMagaluk** [comment]
+1

**Comment by kevinswiber** [comment]
Do changes need to be made to argo/argo-url-router, as well?

**Comment by mdobson** [comment]
Correct. I did guard against the parameter not being passed back properly
though in case it's extended by someone else. That PR is coming shortly.

-Matt

*--*
*Matthew Dobson* | *apigee <https://apigee.com/>* | m: +1.734.634.5472 |
twitter @mdobs <http://twitter.com/mdobs> @apigee
<https://twitter.com/apigee> | *Apigee Community
<http://community.apigee.com/>* for answers, ideas and support!
[image: http://iloveapis.com/]

**Pull request merged**